### PR TITLE
Tutorial videos dependent destroy

### PR DESCRIPTION
**Pull request by rhantak**
- Add dependent destroy to removed all associated videos when a tutorial is destroyed

### CODE REVIEW

File: app/views/admin/dashboard/show.html.erb [hunk 1]
```diff
@@ -12,12 +12,13 @@
...
+      <td><%= link_to tutorial.title, tutorial_path(tutorial) %><br><br><hr></td>
       <td>
-        <%= link_to "Edit", edit_admin_tutorial_path(tutorial) %> |
-        <%= link_to 'Destroy', tutorial_path(tutorial),
-                               method: :delete,
-                               data: { confirm: 'Are you sure?' } %>
+        <%= link_to "Edit", edit_admin_tutorial_path(tutorial) %>
+        <%= button_to 'Destroy', admin_tutorial_path(tutorial),
+                              method: :delete,
+                              data: { confirm: 'Are you sure?' } %>
```
Comment (hillstew) [inline app/views/admin/dashboard/show.html.erb:21]: Oh this is cool 🎉

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -15,6 +15,7 @@
   config.configure_rspec_metadata!
   config.filter_sensitive_data("<YOUTUBE_API_KEY>") { ENV['YOUTUBE_API_KEY'] }
   config.filter_sensitive_data("<USER_GITHUB_TOKEN>") { ENV['USER_GITHUB_TOKEN'] }
+  config.filter_sensitive_data("<EMPTY_USER_TOKEN>") { ENV['EMPTY_USER_TOKEN'] }
```
Comment (hillstew) [inline spec/rails_helper.rb:18]: I had forgot about needing to add this. Nice catch!

File: app/views/admin/dashboard/show.html.erb [hunk 3]
```diff
@@ -12,12 +12,13 @@
   </tr>
   <% @facade.tutorials.each do |tutorial| %>
     <tr class="admin-tutorial-card">
-      <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
+      <td><%= link_to tutorial.title, tutorial_path(tutorial) %><br><br><hr></td>
```
Comment (hillstew) [inline app/views/admin/dashboard/show.html.erb]:
```suggestion
      <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
```
I would rather do this styling with css. What do you think?

**Comment by hillstew** [comment]
This looks good. If you agree with the css comment, just merge that suggestion and I will then merge the PR. Nice job!

File: app/views/admin/dashboard/show.html.erb (hunk 3 above)
Comment (rhantak) [reply app/views/admin/dashboard/show.html.erb]: I totally agree, this was a bit of a quick fix. I'll commit the suggestion!

**Pull request merged**